src/WorkItems/WorkItems.csproj

```diff
@@ -47,4 +47,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0">
```

Comment:
>Microsoft.CodeAnalysis.NetAnalyzers [](http://example.com/codeflow?start=30&length=35)

question, this is new, not an update. What is this package for?
I quickly searched:
[https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisnetanalyzers](https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisnetanalyzers)

I see below:
You do not need to manually install this NuGet package to your project if you are using .NET5 SDK or later. These analyzers are enabled by default for projects targeting .NET5 or later. For projects targeting earlier .NET frameworks, you can enable them in your MSBuild project file by setting one of the following properties:

EnableNETAnalyzers
```
<PropertyGroup>
  <EnableNETAnalyzers>true</EnableNETAnalyzers>
</PropertyGroup>
AnalysisLevel
```
<PropertyGroup>
  <AnalysisLevel>latest</AnalysisLevel>
</PropertyGroup>

Reply:
Excellent feedback! Do you have time to fix everything up and update this PR?

Reply:
You do not need to manually install this NuGet package to your project if you are using .NET5 SDK or later.  ---- we are using 3.1 not 5 and later, so I think no change needed. I noted it down still I think may be a fyi for Ling. 

I will change the CoyoteVersion

Reply:
I actually changed it due to this:
`It is recommended that you enable the analyzers from the .NET SDK instead of installing the Microsoft.CodeAnalysis.NetAnalyzers NuGet package, when possible.` 
[https://learn.microsoft.com/en-us/visualstudio/code-quality/install-net-analyzers?view=vs-2022](https://learn.microsoft.com/en-us/visualstudio/code-quality/install-net-analyzers?view=vs-2022)

Another thing is do we want to use:
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
***with this we will see 671 errors***